.travis.yml

```diff
@@ -1,8 +1,7 @@
 dist: xenial
 branches:
   only:
-    - master
-    - /^\d+\.\d+\.\d+(-\S*)?$/
+    - /^\d+\.\d+\.\d+$/
```

Comment:
only on tags?

Reply:
yes, as effectively every successful build on master is a release you can pick up and drop to prod :)

Reply:
or actually, the master still need to be there as the release process won't get kicked off :)

Reply:
yes, that's my point :)

Reply:
but I've been wrong about the magic of travis before as you well know... so I wasn't sure if you had some new trick :)

Reply:
thankfully no tricks :)